code/modules/food_and_drink/alcohol.dm

```diff
@@ -333,7 +333,7 @@
 	module_research = list("vice" = 100)
 
 /obj/item/reagent_containers/food/drinks/curacao
-	name = "Curacao Liqueur"
+	name = "curacao liqueur"
 	desc = "A bottle of curacao liqueur, made from the dried peels of the bitter orange Lahara."
```

Comment:
```suggestion
	name = "curaçao liqueur"
	desc = "A bottle of curaçao liqueur, made from the dried peels of the bitter orange Lahara."
```

Reply:
Sure. Here's hoping we don't get any encoding issues.

Reply:
utf-8 bruv